shared/audit.py

```diff
@@ -165,7 +165,16 @@ def audit_s3_buckets(findings, region):
                     },
                 )
             )
-
+            
+ def audit_encrypt_s3(findings, region):
```

Comment:
This would be better added as part of the `audit_s3` function which checks the policy and ACL already.

Reply:
I think encryption is totally different part , so we should add it as different function and other information related to encryption should be added there

Reply:
I think we should add this to `audit_s3_buckets` as well. 

IMO it would kind of be nice to break out the body of `audit_s3_buckets` to call something like `check_s3__bucket_acl`/ `check_s3__bucket_policy`/` `check_s3__bucket_encryption`, instead of the e.g. `Check ACL`  comments.